serve.py

```diff
@@ -93,7 +93,7 @@ def _fixup_deprecated_host_args(arguments):
 server_kwargs['websocket_max_message_size'] = 100 * 1024 * 1024
 
 # increase the maximum upload size (default is 100MB)
-server_kwargs['http_server_kwargs'] = {'max_buffer_size': 300 * 1024 * 1024}
```

Comment:
Unfortunately, it is quite relevant for Balloon airframe to take long and detailed logs because the balloon flight typically takes hours and there is a requirement to precisely record vibrations during the flight.

The possible solutions which could reduce review server storage clogging are: 

- Allow large logs for Balloon airframe only
- Detect takeoff and landing of the balloon in the PX4 (I do not know how to that because it is largely unwanted to have lost log data during the balloon flight, which is expensive action), but it is also relevant at the same time because the Industrial SLC SDcards are storage-expensive devices.

Reply:
I see, makes sense. What data rate do you require? The current rate is 250Hz.
If you're interested in vibration, just storing a vibration metric (ekf2 calculates one), or FFT peaks might be an option?

> Allow large logs for Balloon airframe only

That could also be an option

Reply:
I have reverted the size limit to the original 300 MB limit in 4b5d65dd8c5868f9f4e3af9db05ca0104142d586
I have found that has a workaround in the case of local usage by [running the serve.py directly on a large file](https://github.com/PX4/flight_review#usage)